test/fixtures/confidential_simple/network.tf

```diff
@@ -0,0 +1,39 @@
+/**
```

Comment:
In all the other existing fixtures in this repo, `network.tf` is being used as a soft link to `../../shared/network.tf` instead of a new file creation. Can't be used in the same way for this fixture too?

Reply:
I couldn't find a place which networks.tf is being used as a soft link to ../../shared/network.tf, could you please send me an example?

Reply:
Sure, check [instance_template simple fixture file](https://github.com/terraform-google-modules/terraform-google-vm/blob/master/test/fixtures/instance_template/simple/network.tf) for reference. In fact, all the fixtures I checked are using this same soft link.